lib/activity_runtime_api.rb

```diff
@@ -43,19 +43,20 @@ def self.publish_activity(hash, user)
       investigation = Investigation.create(:name => hash["name"], :user => user)
       activity = activity_from_hash(hash, investigation, user)
       external_activity = ExternalActivity.create(
-        :name             => hash["name"],
-        :description      => hash["description"],
-        :abstract         => hash["abstract"],
-        :url              => hash["url"],
-        :thumbnail_url    => hash["thumbnail_url"],
-        :launch_url       => hash["launch_url"] || hash["create_url"],
-        :author_url       => hash["author_url"],
-        :print_url        => hash["print_url"],
-        :template         => activity,
-        :publication_status => "published",
-        :user => user,
-        :author_email => hash["author_email"],
-        :is_locked => hash["is_locked"]
+        :name                   => hash["name"],
+        :description            => hash["description"],
+        :abstract               => hash["abstract"],
+        :url                    => hash["url"],
+        :thumbnail_url          => hash["thumbnail_url"],
+        :launch_url             => hash["launch_url"] || hash["create_url"],
+        :author_url             => hash["author_url"],
+        :print_url              => hash["print_url"],
+        :student_report_enabled => hash["student_report_enabled"],
```

Comment:
This doesn't seem to be handling sequence level student report enabled settings.  In that case I'd expect that in LARA an author would set the student_report_enabled to false on at least one of the activities of the sequence. So then the portal will have look at all of the activities and if any have student_report_enabled false, then the report should also be disabled in the portal.

Reply:
I added a test for that here - https://github.com/concord-consortium/rigse/pull/503/files#diff-42df82be6d7de9f021053e849a8cbf2eR492